src/Altinn.ApiClients.Maskinporten/Config/MaskinportenSettings.cs

```diff
@@ -78,5 +78,10 @@ public class MaskinportenSettings
         /// Optional. Enabels verbose logging that should only be enabled when troubleshooting. Will cause logging (with severity "Information") of assertions.  
         /// </summary>
         public bool? EnableDebugLogging { get; set; }
+
+        /// <summary>
+        /// Optional. Enables renewing of the Bearer JWT, even if it already exists in the Authorization-header
+        /// </summary>
+        public bool? AlwaysRenewToken { get; set; }
```

Comment:
```suggestion
        /// Optional. Overwrites existing Authorization-header if set. Default: ignore existing Authorization-header.
        /// </summary>
        public bool? OverwriteAuthorizationHeader { get; set; }
```

`AlwaysRenewToken` blir upresist, fordi MaskinportenService vil cache tokenet avhengig av levetiden på det.